WikiWatershed/model-my-watershed
# BiG-CZ: WDC / CUAHSI Details View

## Overview

Adds specific items to CUAHSI's details view, based on comments in https://github.com/WikiWatershed/model-my-watershed/issues/2234#issuecomment-328214065 and [this Google doc](https://docs.google.com/document/d/1Kot_Toow6FPpI9FHImsx078W7Jn3UwMME4MOnersX2A). Important to note: this does not fetch the detail information for a given item. That will be done in #2238, which depends on #2243.

Also tagging @aufdenkampe for visual review.

Connects #2234 

### Demo

![image](https://user-images.githubusercontent.com/1430060/30453159-eb30966a-9965-11e7-954e-a7d5b349d581.png)

### Notes

Some fields have placeholder values, such as the citation in the above screenshot. These placeholders seem arbitrary, and to convert them all into real values may be a long-tailed effort. Currently they are left as is.

## Testing Instructions

 * Check out this branch. Run `bundle`
 * Go to [:8000/?bigcz](http://localhost:8000/?bigcz)
 * Search for something and open the WDC tab. Click on any item to get to the Details view.
 * Ensure the values match the description in the Google doc and the accompanying source comment.
